lib/User.rb

```diff
@@ -0,0 +1,211 @@
+require 'pry'
+require './Person'
+require './GroceryList'
+
+class User < Person 
+  attr_accessor :name, :age, :gender
+  def initialize(args = {})
+    super(args)
+      @spaghetti     = args[:spaghetti]
+    # @answers   = args[:answers]
```

Comment:
Better to pass in a GroceryList instance when you initialize it. That way you get access to all the instance variables inside the `GroceryList` class!